backend/src/services/user.go

```diff
@@ -14,9 +14,9 @@ import (
 )
 
 type UserServiceInterface interface {
```

Comment:
```suggestion
type UserServiceInterface interface {
	CreateUser(userBody models.CreateUserRequestBody) (*models.User, *errors.Error)
```
for idiomatic order of CRUD